src/Compositor.cpp

```diff
@@ -1109,6 +1109,20 @@ wlr_surface* CCompositor::vectorToLayerSurface(const Vector2D& pos, std::vector<
     return nullptr;
 }
 
+SIMEPopup* CCompositor::vectorToIMEPopup(const Vector2D& pos, std::list<SIMEPopup>* popups) {
```

Comment:
why not a `const &`?

Reply:
You mean
```suggestion
SIMEPopup* CCompositor::vectorToIMEPopup(const Vector2D& pos, const std::list<SIMEPopup>& popups) {
```
and using `const_cast` to return the pointer? I don't see much use of `const_cast` in the code base.

Reply:
I don't think you need to use const cast?

oh, maybe, cuz it's a list.

Then just & without const. My bad

Reply:
No do not do that it will fail if you compile it with changing it to &

Reply:
?